src/box/sql/os_unix.c

```diff
@@ -1433,14 +1433,16 @@ unixTempFileDir(void)
 		"/tmp",
 		"."
 	};
+	static char zbuf0[PATH_MAX];
+	static char zbuf1[PATH_MAX];
```

Comment:
Nit: Better use `malloc` instead of static buffers - then no extra memory will be occupied unless this function is called.

Reply:
I'd like to, but I failed to find `sql_destroy` or another point where I could place deinitialization code and free these vars.
Maybe make the values shorter? 128 or 256 bytes, for example.

Reply:
Maybe re-check the env values on each `unixTempFileDir` invocation without caching?

Reply:
> I'd like to, but I failed to find `sql_destroy` or another point where I could place deinitialization code and free these vars.

I think it'd be okay to not free them. (AFAIK we don't have a strict requirement to free all data allocated with malloc). Anyway, I'm fine with the static buffer as well: after all, no physical memory will be occupied until the buffer is accessed (only virtual memory is preallocated for static variables).